index.js

```diff
@@ -476,6 +480,10 @@ const getAllGamepads = () => {
     allGamepads = [
       new Gamepad('left', 0),
       new Gamepad('right', 1),
+      new Gamepad('player1', -1),
```

Comment:
Is there a way to expose these only when connected?

Technically the existing implementation is a bug since it doesn’t do that and I’d like to fix it soon.

But this change further cements the existing mistake...